response_operations_ui/views/collection_exercise.py

```diff
@@ -1104,3 +1106,21 @@ def _add_collection_instrument(short_name, period):
 
     form.formtype.data = ""  # Reset the value on successful submission
     return get_view_sample_ci(short_name, period)
+
+
+@collection_exercise_bp.route("/cir", methods=["GET"])
+@login_required
+def get_cir_service_status():
+    logger.info("Get CIR service status")
+    error_message = None
+    response_content = None
+    try:
+        response_content = cir_controller.get_cir_service_status()
+    except HTTPError as e:
+        error_message = str(e)
```

Comment:
Of course we wouldn't normally expose this to the user, and are likely to replace the verbatim exception message with an error code and user friendly message. However, for developing this integration this is fine for the time being.